release/release_tests.yaml

```diff
@@ -3970,6 +3969,8 @@
       cluster:
         cluster_env: stress_tests/state_api_app_config.yaml
         cluster_compute: stress_tests/stress_tests_compute_large_gce.yaml
+      smoke_test:
+        frequency: manual
```

Comment:
what does `smoke_test` mean specifically under this context? why is `frequency` under `smoke_test`?

Reply:
Oh, I didn't add GCE variance for smoke tests (i.e, I didn't create the gce compute config for smoke tests) since it doesn't provide extra signals.

Reply:
The test runner auto-generates smoke tests as they inherit from the default definition. This line override that inheritance to set smoke test frequency to manual ;)

Reply:
Just to point out that the smoke test currently cannot run on GCE since there is no gce compute config.

Reply:
Ack, this change will stop those broken smoke GCE from running in nightly build